applications/callflow/src/cf_attributes.erl

```diff
@@ -204,20 +205,18 @@ maybe_ensure_cid_valid(Number, Name, _, Attribute, Call) ->
                                {api_binary(), api_binary()}.
 maybe_cid_privacy(Number, Name, Call) ->
     case wh_util:is_true(whapps_call:kvs_fetch('cf_privacy', Call))
-            orelse ?CALLER_PRIVACY(whapps_call:custom_channel_vars(Call))
+        orelse ?CALLER_PRIVACY(whapps_call:custom_channel_vars(Call))
     of
         'true' ->
             lager:info("overriding caller id to maintain privacy"),
-            {whapps_config:get_non_empty(
-               <<"callflow">>
-               ,<<"privacy_number">>
-               ,wh_util:anonymous_caller_id_number()
-              )
-             ,whapps_config:get_non_empty(
-                <<"callflow">>
-                ,<<"privacy_name">>
-                ,wh_util:anonymous_caller_id_name()
-               )
+            {whapps_config:get_non_empty(<<"callflow">>
+                                        ,<<"privacy_number">>
```

Comment:
Umm...I just follow what Emacs says :) I'm not sure when it changed indenting by one there...weird.